src/js/page_sections/Menu/Menu.js

```diff
@@ -108,6 +104,7 @@ class Menu extends Component {
               theme={theme}
               handleMenuToggle={this.props.toggleMenu}
               handleSublistToggle={this.toggleSublist}
+              intl={intl}
```

Comment:
instead of passing intl around as a prop can we just inject it where we need it like: 
`const superI18ncomponent = injectIntl(({someprop, intl}) => ( ... ) `

Reply:
In all fairness, I probably set pretty bad precedent for this in Menu.js. Instead of `<AirportMobileListItem intl={intl} />` I should have ` const AirportMobileListItem = injectIntl((props) => ( ... ))`